### PR TITLE
Balancing Exotic Weapons & Sound Additions

### PR DESCRIPTION
**Pull request by Worldwave**
Rebalanced damage values for exotic codenames and the anti-shield missiles Buffed FALLEN DRAGON
Added new animations to ATRAX
Added 4 new sound files

### CODE REVIEW

File: DATA/FX/effects.ini [hunk 1]
```diff
@@ -1520,10 +1520,26 @@ effect_type = EFT_WEAPON_IMPACT
 vis_effect = pi_particle_03_impact
 
 [Effect]
-nickname = pi_particle_03_proj
```
Comment (XxSARGExX) [inline DATA/FX/effects.ini:1523]: pi_particle_03_proj is exchanged with pi_particle_05_flash (pi_particle_03_proj not there anymore)

**Review by XxSARGExX — CHANGES_REQUESTED** [review]
Missing pi_particle_03_proj in effects ini

**Comment by Worldwave** [comment]
Added back

**Review by XxSARGExX — APPROVED** [review]

File: DATA/FX/beam_effects.ini [hunk 2]
```diff
@@ -722,6 +722,28 @@ head_texture = ball
...
+sec_core_color = 0, 0, 190
+sec_outter_color = 0, 0, 255
+tail_color = 0, 0, 180
+head_brightness = 1
+trail_brightness = 0.3
+head_texture = ball
+trail_texture = wide
+flash_size = 3
+
+; ATRAX Xenos Heavy Particle Weapon
```
Comment (erikroe) [inline DATA/FX/beam_effects.ini:745]: Would be nice to have the comments actually at the right spot above the thing, and not below.

File: DATA/FX/effects.ini [hunk 3]
```diff
@@ -1541,6 +1541,28 @@ effect_type = EFT_WEAPON_PROJ
...
+
+; ATRAX Xenos Heavy Particle Weapon
+
+[Effect]
+nickname = pi_particle_05_proj
+effect_type = EFT_WEAPON_PROJ
+vis_effect = pi_particle_05_proj
+vis_beam = pi_particle_05_beam
+
+; ATRAX Xenos Heavy Particle Weapon
```
Comment (erikroe) [inline DATA/FX/effects.ini:1564]: Same with the comment being at the wrong spot

**Pull request merged**